master/internal/user/service.go

```diff
@@ -247,6 +247,7 @@ func (s *Service) postLogin(c echo.Context) (interface{}, error) {
 		cookie := new(http.Cookie)
 		cookie.Name = "auth"
 		cookie.Value = token
+		cookie.Expires = time.Now().Add(24 * 10 * time.Hour)
```

Comment:
@vishnu2kmohan

Reply:
the expiration date would be useful even outside this, I'm moving it to here https://github.com/determined-ai/determined/pull/330

Reply:
@hkang1 Yeah, I'm okay with it being 7 calendar days. 
(I would like for it to be 5 days but I don't feel **too** strongly about it :grin:)

Reply:
I'm good with any number of days as long as it doesn't immediately expire it's good.